### PR TITLE
Expand notes, change formatting, and update priorities

### PR DESCRIPTION
**Pull request by chloecinders**
- Expands the notes to avoid any confusion
- Moves info from priorities to notes
- Adds "Large bots" priority as they are different from verified and unverified bots
- Changes formatting to be more consistent
- Rewords some sentences

### CODE REVIEW

File: experiment-rollout.js [hunk 1]
```diff
@@ -46,33 +46,45 @@ const data = {
...
             {
                 status: 2,
-                name: 'Discord Staff'
+                name: 'Large bots | Criteria unknown | Reservations only'
             },
             {
                 status: 2,
-                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
+                name: 'Discord Staff | Staff and personal accounts'
             },
```
Comment (Tolga1452) [inline experiment-rollout.js]: wdym by "large bots" if they're not verified? also why the order has changed?

File: experiment-rollout.js [hunk 2]
```diff
@@ -46,33 +46,45 @@ const data = {
...
-                status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
+                status: 2,
+                name: 'Partnered / verified / ex-partnered server owners'
             },
             {
-                status: 0,
-                name: 'Top (likely 100, by server count) bot developers (Unconfirmed)'
+                status: 1,
+                name: 'Verified bots | Ordered by support server member count'
```
Comment (Tolga1452) [inline experiment-rollout.js]: where did you get that "support server member count" info?

File: experiment-rollout.js [hunk 3]
```diff
@@ -46,33 +46,45 @@ const data = {
...
         notes: [
             {
-                title: 'Checking Pomelo Usernames',
-                text: 'Checking Pomelo usernames via API has patched and accounts which tried to check usernames will be IP banned from Discord.'
+                title: 'Checking claimed/reserved names',
+                text: 'Checking claimed names can be done through Friend requests. This however, is not recommended as it can lead to your account triggering spam/bot detections and thus getting suspended or banned entirely. Reserved names cannot be checked anymore as the API endpoint has been gated.'
+            },
+            {
+                title: 'Pomelo check bots',
+                text: 'Pomelo check bots operate on either the Friend request api or on lists/databases made before the API endpoint gate. This means that they are inaccurate.'
```
Comment (Tolga1452) [inline experiment-rollout.js]:
```suggestion
                text: 'Pomelo check bots operate on either the Friend request API or on lists/databases made before the API endpoint gate. This means that they are inaccurate.'
```

File: experiment-rollout.js [hunk 4]
```diff
@@ -46,33 +46,45 @@ const data = {
...
+            },
+            {
+                title: 'Pomelo reservations',
+                text: 'Unnamed Discord partners have confirmed that Discord took a snapshot of all usernames and have reserved a bunch of usernames, even for normal users.'
+            },
+            {
+                title: 'Bot name conflicts',
+                text: 'if two or more bots have the same name the bot with the least members in its support server will get the name. The other ones will get their discriminator appended to their name. I.e. @bot1234'
```
Comment (Tolga1452) [inline experiment-rollout.js]:
```suggestion
                text: 'If two or more bots have the same name the bot with the least members in its support server will get the name. The other ones will get their discriminator appended to their name. I.e. @bot1234'
```

also "least members in its support server" again?

File: experiment-rollout.js (hunk 1 above)
Comment (chloecinders) [reply experiment-rollout.js]: There was another priority for large bots, i.e. Dyno and Mee6 who got their names reserved but not yet changed. This was also directly after brands and that's why the order changed.

**Comment by evnsh** [comment]
Hello!

Could you elaborate on the following:
> Brands | Reserved unless Discord partner

Discord Partner aren't able to choose the username that they want, they have a specific username that is reserved to them, generally a variation of their current username and aren't able to claim brand names which are blacklisted by Discord.

> Large bots | Criteria unknown | Reservations only

Large bots would imo fall into the Brand category, and so far no "large bots" have been able to claim their username (most of them are however reserved and cannot be picked by another user)

> Verified bots | Ordered by support server member count

Where does the support server member count come from? Again large bots like MEE6 or YAGPDB have not been given access to Pomelo yet.

> Everyone else | Ordered by Nitro/Account creation

Why did you remove the March 1st date for Nitro? This seems pretty important imo.

File: experiment-rollout.js (hunk 2 above)
Comment (chloecinders) [reply experiment-rollout.js]: Thats from the "Server count wins name conflicts, other ones will get their discriminator appended", reworded

File: experiment-rollout.js (hunk 1 above)
Comment (Tolga1452) [reply experiment-rollout.js]: ah okay got it

File: experiment-rollout.js (hunk 2 above)
Comment (Tolga1452) [reply experiment-rollout.js]: but "server count", not "member count in support server"

Comment (chloecinders) [reply experiment-rollout.js]: I misinterpreted the original priority name - fixed

File: experiment-rollout.js (hunk 4 above)
Comment (chloecinders) [reply experiment-rollout.js]: fixed

File: experiment-rollout.js [hunk 5]
```diff
@@ -46,33 +46,53 @@ const data = {
...
             },
             {
                 status: 2,
-                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
+                name: 'Partnered / verified / ex-partnered / server subscription server owners'
             },
             {
                 status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
+                name: 'Verified bots | Ordered by guild count'
```
Comment (Tolga1452) [inline experiment-rollout.js]:
```suggestion
                name: 'Verified bots | Ordered by server count'
```

"server" can be clearer

**Comment by chloecinders** [comment]
> Hello!

Hey!
 
> Could you elaborate on the following:
> 
> > Brands | Reserved unless Discord partner
>
> Discord Partner aren't able to choose the username that they want, they have a specific username that is reserved to them, generally a variation of their current username and aren't able to claim brand names which are blacklisted by Discord.

Yep, that's why I added "Reserved unless Discord partner". Brands like Youtube and TikTok have their names reserved, not claimed. Other brands also had their names reserved but due to having a Partnered brand account (i.e. the brand account is a server owner) they could claim their name.

> > Large bots | Criteria unknown | Reservations only
> 
> Large bots would imo fall into the Brand category, and so far no "large bots" have been able to claim their username (most of them are however reserved and cannot be picked by another user)

Agreed. Included it in the brand category. Also the "Reservations only" again, bot developers have come out and said that they can reserve a name they want. They cant claim it though. That's why I added the Reservations only.

> > Verified bots | Ordered by support server member count
> 
> Where does the support server member count come from? Again large bots like MEE6 or YAGPDB have not been given access to Pomelo yet.

Changed to Ordered by guild count.

> > Everyone else | Ordered by Nitro/Account creation
> 
> Why did you remove the March 1st date for Nitro? This seems pretty important imo.

Added it back into notes.

File: experiment-rollout.js [hunk 6]
```diff
@@ -46,33 +46,53 @@ const data = {
...
             },
             {
                 status: 0,
-                name: 'Top (likely 100, by server count) bot developers (Unconfirmed)'
-            },
```
Comment (Tolga1452) [inline experiment-rollout.js]: where did this go?

**Review by Tolga1452 — CHANGES_REQUESTED** [review]

File: experiment-rollout.js (hunk 6 above)
Comment (chloecinders) [reply experiment-rollout.js]: Its unconfirmed, so I thought it was unnecessary? People take that stuff as face value and spread it as if it were 100% confirmed by Discord staff.

Comment (Tolga1452) [reply experiment-rollout.js]: yeah keeping it untill 100% confirmation will be better

**Review by Tolga1452 — APPROVED** [review]
i think it's ready to merge now?

**Comment by evnsh** [comment]
> Partnered / verified / ex-partnered / server subscription server owners

Not every ex-partner gets Pomelo, this needs to be clearer.

> Everyone else | Ordered by Nitro/Account creation

Please put back the 1st March date for Nitro it's important.

Rollout to Verified Bots hasn't started afaik, the status should be 0.

**Comment by Tolga1452** [comment]
> Please put back the 1st March date for Nitro it's important.

done ✅

**Comment by Tolga1452** [comment]
> Rollout to Verified Bots hasn't started afaik, the status should be 0.

you set it in the pr lol. but i can confirm it hasn't started yet

**Comment by chloecinders** [comment]
Changed status to 0

**Review by Tolga1452 — APPROVED** [review]

**Pull request merged**